lib/galaxy_test/selenium/test_collection_edit.py

```diff
@@ -13,6 +13,10 @@ def test_change_dbkey_simple_list(self):
         self.create_simple_list_collection()
         self.open_collection_edit_view()
         self.navigate_to_database_tab()
+        assert (
+            "This will create a new collection in your History. Your quota will not increase."
+            in self.find_element_by_selector("div.alert-info").text
```

Comment:
Can you place `div.alert-info` in navigation.yml, then use `self.componentns.<your_fancy_new_selector>.wait_for_visible()`, and only then assert the text in the element ? That should be a bit more robust.